### PR TITLE
ref(ratelimit): Change RedisRatelimiter to use redis cluster

### PR DESCRIPTION
**Pull request by AniketDas-Tekky**
Should make the redis rate limiter more robust to node failures and removes another reference to rb

### CODE REVIEW

File: src/sentry/ratelimits/redis.py [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
 
 class RedisRateLimiter(RateLimiter):
     window = 60
 
     def __init__(self, **options):
-        self.cluster, options = get_cluster_from_options("SENTRY_RATELIMITER_OPTIONS", options)
+        cluster_key = getattr(settings, "SENTRY_RATE_LIMIT_REDIS_CLUSTER", "default")
+        self.client = redis.redis_clusters.get(cluster_key)
 
     def validate(self):
```
Comment (wedamija) [inline src/sentry/ratelimits/redis.py:20]: I don't actually see any calls to this at all, I think it can be removed

**Review by wedamija — APPROVED** [review]
lgtm, probably worth checking that ops is fine with this going on the default redis cluster before we merge

File: src/sentry/ratelimits/redis.py (hunk 1 above)
Comment (untitaker) [reply src/sentry/ratelimits/redis.py:20]: The validate function is a service class thing iirc

**Review by beezz — CHANGES_REQUESTED** [review]
lgtm, but merging needs to wait for cluster setup in production

**Comment by AniketDas-Tekky** [comment]
OPS request to add default redis cluster: https://getsentry.atlassian.net/browse/OPS-1143

**Review by beezz — APPROVED** [review]
🚀

**Pull request merged**